ext_conf_template.txt

```diff
@@ -0,0 +1,8 @@
+# cat=basic/enable; type=boolean; label=use flux form groups: If enabled, grouping the select items from the fluid content type field with the name of the wizard tab element.
```

Comment:
useGroups makes really good sense - nice addition! So appropriate in fact that I think this should simply be the default and needs no way to disable it ;)

But `showIconTable` and `showIconTableCols` should just be TCA options (which can be configured with AdditionalConfiguration, with overrides, etc.).

If we change the behavior to always group, and preserve the icon table options as only TCA, we can avoid extension configuration completely.